.docs/README.md

```diff
@@ -12,6 +12,7 @@
     - [Security](#securityconsole)
     - [Utils](#utilsconsole)
     - [Advanced cache](#advancedcacheconsole)
+- [Kdyby/Console compatibiĺity](#kdybyconsole-compatibility)
```

Comment:
I would rename it to 3rd party or external or compatiblity and moved kdyby/console under this headline.

Reply:
Updated, how about now? :)

Reply:
I think you could also drop list of available commands, it is duplicate

Reply:
Agreed. Except that, I'm going to merge it ;-)

Reply:
Removed :)